### PR TITLE
J24BTW-6:Back-End Setup: Knex Migrations and Seeds

### PR DESCRIPTION
**Pull request by trupti-patil23**
Tim, Can you please review.

### CODE REVIEW

File: .env.example
Comment (timho1985) [inline .env.example:1]: Shall we keep this file and update it to include database related variables?

Comment (trupti-patil23) [reply .env.example:1]: I think lets keep .env file . Developer can change .env as per their settings

Comment (trupti-patil23) [reply .env.example:1]: I have changed to .env.example

**Review by timho1985 — APPROVED** [review]
It looks good to me

**Pull request merged**